sky/exceptions.py

```diff
@@ -132,6 +132,13 @@ class ClusterSetUpError(Exception):
     pass
 
 
+class ClusterDoesNotExist(ValueError):
+    """Raise when trying to operate on a cluster that does not exist."""
+    # This extends ValueError for compatibility reasons - we used to throw
+    # ValueError instead of this.
+    pass
```

Comment:
@Michaelvll Previously we were using ValueError, so the new exception also extends ValueError. This means that any existing code that catches ValueError should also catch this. Same goes for user code using the python API.